lib/ui/views/liquidity_add/layouts/components/liquidity_add_confirm_sheet.dart

```diff
@@ -75,15 +81,50 @@ class LiquidityAddConfirmSheetState
             labelBtn: AppLocalizations.of(context)!.btn_confirm_liquidity_add,
             disabled: !consentChecked && liquidityAdd.consentDateTime == null,
             onPressed: () async {
+              if (liquidityAdd.token1 != null &&
+                  liquidityAdd.token1!.isUCO &&
+                  (Decimal.parse(
```

Comment:
That "double to decimal conversion, then comparison to trigger value" logic should be factorized.

We could create a dedicated family provider.